CHANGELOG.md

```diff
@@ -77,6 +77,8 @@ the `configuration.konghq.com` API group.
   credentials secrets on update of secrets, and on create or update of
   KongConsumers.
   [#729](https://github.com/Kong/kubernetes-ingress-controller/issues/729)
+- Fixed a race condition when creating Enterprise workspaces.
```

Comment:
Given my understanding of this issue, I think it's possible this log entry could be more specific:

```suggestion
- Fixed a race condition where multiple actors may simultaneously
  attempt to create the default Enterprise workspace.
```

Reply:
Rolled into 779c368c